packages/forma-36-website/package.json

```diff
@@ -44,6 +44,6 @@
   "scripts": {
     "build:website": "gatsby build",
     "develop": "gatsby develop",
```

Comment:
Maybe we would then just remove `develop` command and keep `start`? WDYT?

Reply:
yep, will update this